bcipy/signal/tests/model/integration_test_input/et/stimuli_positions.json

```diff
@@ -0,0 +1,120 @@
+{

```

Comment:
This file may be obsolete since we obtain the stimuli position information from each data folder in the newer versions of the data collection

Reply:
I just realized this is only for integration tests, nevermind :)